wordpress-mobile/WordPress-iOS
# Change Gutenberg switch to only affect new posts

See https://github.com/wordpress-mobile/gutenberg-mobile/issues/565

To test:

* Open a post with blocks, it should open in Gutenberg regardless of opt-in
* Open a post without blocks, it should open in Aztec regardless of opt-in
* Create a new post, it should open in Gutenberg if the setting is on or Aztec if the setting is off

I've added a few unit tests to test that behavior as well.

Update release notes:

☝️ We'll add a single entry for all Gutenberg related changes https://github.com/wordpress-mobile/gutenberg-mobile/issues/584

cc @hypest
